components/reusable-components/navigation/navigation.module.scss

```diff
@@ -0,0 +1,62 @@
+.navbar {
+  background-color: var(--bg-color);
+}
+
+.navbarBrand {
+  color: #ffff;
+  font-size: 36px;
+  margin-left: 20px;
+  width: 227px;
+  height: 44px;
+  top: 67px;
+  left: 85px;
+}
+
+.navLink1 {
+  color: #ffff;
+  font-size: 25px;
+  margin-right: -370px;
```

Comment:
We should avoid negative margins, can you try and solve this differently and use the negative margin as a last resort?